extensions/map-layers-formats/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "name": "@itwin/map-layers-formats",
+  "version": "3.4.0-dev.7",
+  "description": "Enables additional map-layers formats in iTwin.js",
+  "main": "lib/cjs/map-layers-formats.js",
+  "module": "lib/esm/map-layers-formats.js",
+  "typings": "lib/cjs/map-layers-formats",
+  "license": "MIT",
+  "scripts": {
+    "build": "npm run -s build:cjs",
+    "build:ci": "npm run -s build && npm run -s build:esm",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "cover": "nyc npm -s test",
+    "docs": "",
```

Comment:
Your beta APIs belong in the docs.

Reply:
Added docs script, doc groups, package description. Added package to doc site layouts.